cmd/buildah/main.go

```diff
@@ -61,6 +61,14 @@ func main() {
 			Usage: "storage driver option",
 			Value: defaultStoreDriverOptions,
 		},
+		cli.StringSliceFlag{
+			Name:  "userns-uid-map",
+			Usage: "default `ctrID:hostID:length` UID mapping to use",
+		},
```

Comment:
Should we make these hidden options so we don't need to document?

Reply:
It's tempting (the language in the man pages about how the mapping options interact could stand to be simplified, but it's about as simple as I can make it), but I'm planning to have `buildah` itself use this in a later PR, and I think we'd be obliged to document it then.